web/ASTU-web-group-2/banking-dashboard/app/components/lastTransaction/LastTransaction.tsx

```diff
@@ -14,7 +14,7 @@ export type Item = {
 
 const LastTransaction = () => {
   const access =
-    "eyJhbGciOiJIUzM4NCJ9.eyJzdWIiOiJzYW1pdGVzdCIsImlhdCI6MTcyMzgxMDA0NiwiZXhwIjoxNzIzODk2NDQ2fQ.6W4MhXDf2fYFVvHonw5Bs597XszBLvNnB71B6pqrbhkMU3IFkF7NLwHLLJPotY51";
+    "eyJhbGciOiJIUzM4NCJ9.eyJzdWIiOiJzYW1pdGVzdCIsImlhdCI6MTcyMzgzNjg1NywiZXhwIjoxNzIzOTIzMjU3fQ.O6YWMCTSeqp9WJXya9rOHpYcCtglFH6T3Ze4W0zpWOGjD0tCUvCv5I-M-b14rdim"; //user another access token if it expires.
```

Comment:
This should be added to the .env file and .gitignore to avoid vulnerability issues, or removed after testing.

Reply:
Fixed! Moved access token to local env